test/functional/fixtures/es_archiver/logstash_functional/mappings.json

```diff
@@ -1088,4 +1097,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Not sure if it matters, but https://github.com/pickypg/kibana/blob/12da6e8029c7d10df0a63d5347a403645d4d4bf0/test/functional/fixtures/es_archiver/logstash_functional/mappings.json#L755 still identifies its type as "docs"